helper/metricsutil/wrapped_metrics.go

```diff
@@ -49,6 +49,23 @@ type Metrics interface {
 
 var _ Metrics = &ClusterMetricSink{}
 
+type SinkWrapper struct {
+	metrics.MetricSink
+}
+
+func (s SinkWrapper) AddDurationWithLabels(key []string, d time.Duration, labels []Label) {
```

Comment:
These seem to be unused.

Reply:
I believe they are necessary to meet the interface requirements for `Metrics`